### PR TITLE
Museum location for Xedra Evolved

### PR DESCRIPTION
**Pull request by Maleclypse**
#### Summary
Mods "Museum Location for Xedra Evolved"

#### Purpose of change
I'm adding more locations to Xedra Evolved that I can migrate the spell learning items drops to.  This will add specific locations that players can look for to find Xedra spells and stuff.

#### Describe the solution
First is a Museum of Curiosities. It will have nested chunks that contain various displays which may contain the spells.   While looking at museums I realized that 1. Museums currently just use alarmed glass rather than alarmed museum glass (acrylic) and 2. All of the horizontal glass walls are unalarmed.  Which is just stupid and gamey.  So I alarmed all of the glass.  Lost the commit with museum glass somehow but decided it was overkill to the solution needed.   Added nests and made changes so that it's clear when you find one instead of the other.   

#### Describe alternatives you've considered
None

#### Testing
tbd

### CODE REVIEW

**Comment by GuardianDll** [comment]
artifacts are so open to public people can do a museum out of them?

**Comment by Maleclypse** [comment]
> artifacts are so open to public people can do a museum out of them?

Most of the displays will be oddities that don't have powers.  The idea is that these are private museums of things taken from the dimensional spillage zones.

File: data/mods/Xedra_Evolved/itemgroups/xe_musuem_itemgroups.json [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+      [ "collarpin", 5 ],
+      [ "postman_shirt", 5 ],
+      [ "halter_top", 50 ],
+      [ "linuxtshirt", 10 ],
+      [ "rashguard", 10 ],
+      [ "sleeveless_tunic", 5 ],
+      [ "poncho", 20 ]
+    ]
+  }
+]
```
Comment (Maleclypse) [inline data/mods/Xedra_Evolved/itemgroups/xe_musuem_itemgroups.json:32]:
```suggestion
]

```

File: data/mods/Xedra_Evolved/snippets/museum.json [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+        "text": "These items were worn by an unidentified man who walked out of a swamp with amnesia after an eclipse."
+      },
+      {  "id": "museum_plaques_visitors_2", "text": "Missing person reappeared in the alleyway they disappearred from twelve years before, they appeared to have only aged a few of those years." },
+      {
+        "id": "museum_plaques_visitors_3",
+        "text": "A woman went missing for six hours.  During the next 7 hours an elderly woman attempted to convince people there, that she was the missing woman.  DNA evidence suggests she is a half sister or aunt to the much younger woman.  No family records suggest this relation exists."
+      }
+    ]
+  }
+]
```
Comment (Maleclypse) [inline data/mods/Xedra_Evolved/snippets/museum.json:47]:
```suggestion
]

```

File: data/mods/Xedra_Evolved/items/clothes.json [hunk 3]
```diff
@@ -8,5 +8,66 @@
...
+      },
+      {
+        "id": "rollerball_fan_tshirt",
+        "name": { "str": "rollerball t-shirt" },
+        "description": "A short-sleeved cotton shirt with the logo of a rollerball team, whatever that is, printed on its front and back.",
+        "weight": 1
+      },
+      {
+        "id": "sunnyvale_tshirt",
+        "name": { "str": "sunnyvale t-shirt", "str_pl": "sunnyvale t-shirts" },
```
Comment (Maleclypse) [inline data/mods/Xedra_Evolved/items/clothes.json:59]:
```suggestion
        "name": { "str": "sunnyvale t-shirt" },
```

File: data/mods/Xedra_Evolved/snippets/museum.json [hunk 4]
```diff
@@ -0,0 +1,47 @@
+[
+  {
+    "type": "snippet",
+    "category": "museum_plaques_items",
+    "text": [
+      {
+        "id": "museum_plaques_items_1",
+        "text": "Discovered in an estate sale in upper Chappaquaddick these baubels were labeled as the owners most precious possessions and proof of other worlds."
```
Comment (Maleclypse) [inline data/mods/Xedra_Evolved/snippets/museum.json:8]:
```suggestion
        "text": "Discovered in an estate sale in upper Chappaquaddick these baubles were labeled as the owners most precious possessions and proof of other worlds."
```

File: data/mods/Xedra_Evolved/snippets/museum.json [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+    "type": "snippet",
+    "category": "museum_plaques_visitors",
+    "text": [
+      {
+        "id": "museum_plaques_visitors_1",
+        "text": "These items were worn by an unidentified man who walked out of a swamp with amnesia after an eclipse."
+      },
+      {
+        "id": "museum_plaques_visitors_2",
+        "text": "Missing person reappeared in the alleyway they disappearred from twelve years before, they appeared to have only aged a few of those years."
```
Comment (Maleclypse) [inline data/mods/Xedra_Evolved/snippets/museum.json:39]:
```suggestion
        "text": "Missing person reappeared in the alleyway they disappeared from twelve years before, they appeared to have only aged a few of those years."
```

**Pull request merged**